File: pset6/mario.py
import cs50

def main():
    while True:
        print("Please indicate the height of the half-pyramid")
        h = cs50.get_int()
        if h > 0 and h < 24:
            break
    build_pyramid(h)
    
def build_pyramid(h):
    r = 1
    for r in range(h):
        for s in range(h + 1 - r):
            print(" ", end = "")
        for n in range(1 + r):
            print("#", end = "")
        print(" ", end = "")
        for f in range(1 + r):
            print("#", end = "")
        print("")
        
if __name__ == "__main__":
    main()

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>   

int main(void)
{
    printf("Please indicate the height of the half-pyramid\n");
    int height;
    height = get_int();
    while (height<0 || height>23)
    {
        printf("Please indicate the height of the half-pyramid\n");
        height = get_int();
    }
    for (int row =1; row<height+1; row++)
    {
        for (int s=0; s<height-row; s++)
        {
        printf(" ");
        }
        for (int h=0; h<1+row; h++)
        {
        printf("#");
        }
        printf("\n");
        
    }
}